CHANGELOG.md

```diff
@@ -1,17 +1,117 @@
 This is the changelog for the Flux daemon; the changelog for the Helm
 operator is in [./CHANGELOG-helmop.md](./CHANGELOG-helmop.md).
 
+## 1.12.0 (2019-04-11)
+
+This release renames some fluxctl commands and arguments while
```

Comment:
Is PR #1777 intentionally omitted in the list below?

Reply:
After having made it the headline, I forgot to put it as a line item :flushed: